test/unit/puppet_fact_parser_test.rb

```diff
@@ -2,6 +2,7 @@
 
 class PuppetFactsParserTest < ActiveSupport::TestCase
   attr_reader :importer
+  attr_accessor :importer
```

Comment:
I can't see any use of the writer?

accessor also obsoletes the need for a reader on line 4.